### PR TITLE
fix(tooltip): show tooltip on hover

### LINKED ISSUE #233: [Tooltip] stopped working
## Bug report

tooltip does not react on hover

Image

### Expected behavior

tooltip should get shown on hover

### PR DESCRIPTION
**Pull request by pawel-twardziak**
Closes #233

### CODE REVIEW

**Comment by pawel-twardziak** [comment]
@pimenovoleg this one is based on a branch of that PR https://github.com/radix-ng/primitives/pull/232 so it must wait until #232 gets marged

File: packages/primitives/tooltip/src/tooltip-root.directive.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export class RdxTooltipRootDirective implements OnInit {
     /**
      * The controlled open state of the tooltip. Must be used in conjunction with onOpenChange.
      */
-    readonly open = input<boolean | undefined, BooleanInput>(false, { transform: booleanAttribute });
+    readonly open = input<boolean | undefined, BooleanInput>(void 0, { transform: booleanAttribute });
```
Comment (pawel-twardziak) [inline packages/primitives/tooltip/src/tooltip-root.directive.ts:73]: this fix is relevant, all the rest comes from PR #232 - that base PR must get merged before this one

Comment (pawel-twardziak) [inline packages/primitives/tooltip/src/tooltip-root.directive.ts:73]: this fix is relevant, all the rest comes from PR #232 - that base PR must get merged before this one

File: packages/primitives/popover/stories/utils/with-option-panel.component.ts [hunk 2]
```diff
@@ -99,26 +107,36 @@ export class WithEventBaseComponent implements AfterContentInit {
...
-        if (this.containers.length > 1) {
-            if (isDevMode()) {
-                console.error('<story>.elementRef.nativeElement', this.elementRef.nativeElement);
-                console.error('<story>.containers', this.containers);
-                throw Error('each story should have only one container!');
-            }
-        }
-        this.paramsContainers = Array.from(this.elementRef.nativeElement?.querySelectorAll('.ParamsContainer') ?? []);
+    constructor() {
+        afterNextRender({
```
Comment (pimenovoleg) [inline packages/primitives/popover/stories/utils/with-option-panel.component.ts:111]: [afterNextRender](https://angularexperts.io/blog/new-angular-coding-style)

Here is a suggestion to use this coding style:

```
@Component({ /* ... */ })
export class MyComponent {
  #logProductChanges = effect(() => { /* ... */
  });
  #afterNextRenderRef = afterRender(() => { /* ... */
  });
  #afterNextRender = afterNextRender(() => { /* ... */
  });
}

```

Comment (pawel-twardziak) [reply packages/primitives/popover/stories/utils/with-option-panel.component.ts]: @pimenovoleg yes, this is something I also like but on the other hand we don't use references anywhere in the code so they are kinda useless. I am more into following the KISS and YAGNI rules - if you don't use it, we don't need it. It is also a matter of preferences in this case IMHO :) one like this while others like that. My preference is:

- I assign the reference when I need to call it
- typically, I create a method wrapper for the stuff and call it within the constructor

What do you think about that approach @pimenovoleg ? Of course,  I can align the code if we have strict rules for such situations :)

Comment (pimenovoleg) [reply packages/primitives/popover/stories/utils/with-option-panel.component.ts]: I just had thoughts :) I agree that there are questions about this approach. We will wait for the styles update from the Angular team.

**Review by pimenovoleg — APPROVED** [review]

File: packages/primitives/popover/stories/utils/with-option-panel.component.ts (hunk 2 above)
Comment (pawel-twardziak) [reply packages/primitives/popover/stories/utils/with-option-panel.component.ts]: Ok, cool, I am merging :)

**Pull request merged**